spec/index.bs

```diff
@@ -27,6 +27,73 @@ Introduction {#intro}
 
 This specification rocks.
 
+Security considerations {#security}
+=====================
+
+<dfn dfn>physical resource</dfn> TEMP
```

Comment:
Maybe we should make a section to put all our temporary dfn dfns in :)

Reply:
huh, that was supposed to be removed once we land #456 :)